iris/src/iris/domain/status/global_search_status_update_dto.py

```
@@ -0,0 +1,17 @@
+
+class GlobalSearchStatusUpdateDTO(StatusUpdateDTO):
+    """Status update DTO for the global search pipeline.
+
+    Sent to Artemis via webhook at two points:
+      1. Immediately after intent classification — answer=None, sources=[] (thinking)
+      2. After the pipeline finishes — answer=str|None, sources=[...]
+    """
+
+    result: Optional[str] = None
+    answer: Optional[str] = None
+    sources: List[LectureSearchResultDTO] = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)